resources/views/components/button/link.blade.php

```diff
@@ -0,0 +1,3 @@
+<button {{ $attributes->merge(['class' => 'underline text-ct-inactive text-ct-start']) }}>
```

Comment:
text-ct-start? do you mean text-left? that one doesn't have to be prefixed with-ct